### PR TITLE
Update backup relayer 

### PR DESCRIPTION
**Pull request by wanglonghong**
## Type of change

- [ ] Docs change / dependency upgrade
- [ ] Configuration / tooling changes
- [ ] Refactoring
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires changes in customer code

## Related Issue(s)

Fixes

### CODE REVIEW

File: packages/utils/src/peripherals/gelato.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ import { BigNumber } from "ethers";
 import { Logger } from "../logging/logger";
 import { jsonifyError } from "../types";
 
-export const GELATO_SERVER = "https://relay.gelato.digital";
+// export const GELATO_SERVER = "https://relay.gelato.digital";
+export const GELATO_SERVER = "https://relay.gelato.digital2";
```
Comment (rhlsthrm) [inline packages/utils/src/peripherals/gelato.ts:8]: We should just allow an override through the config similar to the other relayer.

**Review by rhlsthrm — COMMENTED** [review]
The best way to finish this is the following:
- Make relayers an array in the config. Each relayer can have the following properties: `apiKey`, `url` (is optional, not used for Gelato), `type` (should be `gelato` or `connext` for now).
- Take an array of relayers in the `sendWithRelayerWithBackup` function.

File: packages/adapters/relayer/src/index.ts [hunk 2]
```diff
@@ -41,69 +42,54 @@ export const sendWithRelayerWithBackup = async (
...
-      logger,
-      requestContext,
-    );
-    const status = await relayer.waitForTaskCompletion(taskId, logger, _requestContext);
-    if (status === RelayerTaskStatus.ExecSuccess) {
-      logger.info("Successfully executed fast with primary relayer", requestContext, methodContext, { taskId });
-      return { taskId, taskStatus: status };
+    });
+    try {
+      const taskId = await relayer.instance.send(
```
Comment (rhlsthrm) [inline packages/adapters/relayer/src/index.ts:61]: Log before starting.

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: packages/agents/lighthouse/src/config.ts [hunk 3]
```diff
@@ -39,9 +39,14 @@ export const NxtpLighthouseConfigSchema = Type.Object({
   cartographerUrl: Type.String({ format: "uri" }),
   mode: TModeConfig,
   polling: TPollingConfig,
-  gelatoApiKey: Type.String(),
+  relayers: Type.Array(
+    Type.Object({
+      type: Type.Union([Type.Literal("Primary"), Type.Literal("Backup")]),
```
[the remark this answers is not in the document]
Comment (wanglonghong) [reply packages/agents/lighthouse/src/config.ts:44]: hmm I think we can replace primary/backup with Gelato/Connext

File: packages/adapters/relayer/src/index.ts (hunk 2 above)
Comment (wanglonghong) [reply packages/adapters/relayer/src/index.ts:62]: we have a log before starting.

```typescript
  for (const relayer of relayers) {
    logger.info(`Sending tx with ${relayer.type} relayer`, requestContext, methodContext, {
      chainId,
      domain,
      destinationAddress,
      data,
    });
    try {
      const taskId = await relayer.instance.send(
        chainId,
        domain,
        destinationAddress,
        data,
        relayer.apiKey,
        chainReader,
        logger,
        requestContext,
      );
```

File: packages/agents/lighthouse/src/tasks/processFromRoot/processFromRoot.ts [hunk 4]
```diff
@@ -46,9 +46,9 @@ export const makeProcessFromRoot = async () => {
     context.adapters.database = await getDatabase(context.config.database.url, context.logger);
     for (const relayerConfing of context.config.relayers) {
       const setupFunc =
-        relayerConfing.type == RelayerType.Primary
+        relayerConfing.type == RelayerType.Gelato
```
Comment (rhlsthrm) [inline packages/agents/lighthouse/src/tasks/processFromRoot/processFromRoot.ts:49]: Typo: `relayerConfing` should be `relayerConfig`

**Review by rhlsthrm — CHANGES_REQUESTED** [review]

File: packages/agents/lighthouse/src/tasks/processFromRoot/processFromRoot.ts (hunk 4 above)
Comment (wanglonghong) [reply packages/agents/lighthouse/src/tasks/processFromRoot/processFromRoot.ts]: OMG!

**Review by rhlsthrm — APPROVED** [review]

**Pull request merged**